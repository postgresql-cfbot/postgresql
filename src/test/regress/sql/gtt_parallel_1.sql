

set search_path=gtt,sys;

select nextval('gtt_with_seq_c2_seq');

insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
commit;
select * from gtt1 order by a;

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
rollback;
select * from gtt1 order by a;

truncate gtt1;
select * from gtt1 order by a;

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
truncate gtt1;
select * from gtt1 order by a;
insert into gtt1 values(1, 'test1');
rollback;
select * from gtt1 order by a;

begin;
select * from gtt1 order by a;
truncate gtt1;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
truncate gtt1;
commit;
select * from gtt1 order by a;

reset search_path;

