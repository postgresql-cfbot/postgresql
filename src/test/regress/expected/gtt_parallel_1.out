set search_path=gtt,sys;
select nextval('gtt_with_seq_c2_seq');
 nextval 
---------
       1
(1 row)

insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
 a |   b   
---+-------
 1 | test1
(1 row)

commit;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
 a |   b   
---+-------
 1 | test1
(1 row)

rollback;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

truncate gtt1;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

begin;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
 a |   b   
---+-------
 1 | test1
(1 row)

truncate gtt1;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

insert into gtt1 values(1, 'test1');
rollback;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

begin;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

truncate gtt1;
insert into gtt1 values(1, 'test1');
select * from gtt1 order by a;
 a |   b   
---+-------
 1 | test1
(1 row)

truncate gtt1;
commit;
select * from gtt1 order by a;
 a | b 
---+---
(0 rows)

reset search_path;
